mlir/include/mlir/Dialect/MemRef/Utils/MemRefUtils.h

```diff
@@ -113,4 +119,4 @@ MemrefValue skipViewLikeOps(MemrefValue source);
 } // namespace memref
 } // namespace mlir
 
-#endif // MLIR_DIALECT_MEMREF_UTILS_MEMREFUTILS_H
+#endif // MLIR_DIALECT_MEMREF_UTILS_MEMREFUTILS_H
```

Comment:
The message does not show up in UI, but I think it means that you're missing a new line at the end of the file.

<img width="462" alt="image" src="https://github.com/user-attachments/assets/27c7b09c-ebe8-4321-9d9b-08baba51ec42">